### DISCUSSION TITLE
Safari shows error after first response is done

### DISCUSSION THREAD

**Discussion by PeterHagen** [open-webui/open-webui#4123] [General]
# Bug Report

## Description

**Bug Summary:**

After the first response is done, the UI doesn't becomes sort of unresponsive. It's still possible to type further, but no response will be shown. On reload of the page, messages with `[object Object]` are shown. This behaviour occurs since the last update.

**Steps to Reproduce:**

 start a chat with any LLM and send first chat message

**Expected Behavior:**

Normal chat behaviour where you can keep on chatting

**Actual Behavior:**

UI seems to become inresponsive. No further chat occurs

## Environment

- **Open WebUI Version:** v0.3.10
- **Ollama (if applicable):** 0.2.8

- **Operating System:** macOS 14.5
- **Browser (if applicable):** Safari 17.5

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**

```
[Log] {"model":"gemma2:latest","created_at":"2024-07-25T19:48:22.626082Z","message":{"role":"assistant","content":"."},"done":false} (Help.b5c10dca.js, line 153)
[Log] {"model":"gemma2:latest","created_at":"2024-07-25T19:48:22.649381Z","message":{"role":"assistant","content":""},"done_reason":"stop","done":true,"total_duration":7167163083,"load_duration":49374541,"prompt_eval_count":115,"prompt_eval_duration":503887000,"eval_count":290,"eval_duration":6610776000} (Help.b5c10dca.js, line 153)
[Error] Failed to load resource: the server responded with a status of 500 () (completed, line 0)
[Log] SyntaxError: The string did not match the expected pattern. — index.ts:87 (index.c405526f.js, line 1)
[Error] Unhandled Promise Rejection: TypeError: DOMException is not a constructor (evaluating 'new t(e)')
	ut (scheduler.8ceb707f.js:1:8554)
```

**Screenshots (if applicable):**

<img width="966" alt="Screenshot 2024-07-25 at 21 49 43" src="https://github.com/user-attachments/assets/3dc12e0b-8092-4a20-9edd-dc028c45f616">

## Installation Method

Open-webui runs in Docker, with Ollama installed on the host

**Comment by PeterHagen** [comment]
Additional screenshot with details error message

<img width="1098" alt="Screenshot 2024-07-25 at 22 03 36" src="https://github.com/user-attachments/assets/007197d6-15ac-430f-abbd-da9e23ad8f26">

**Comment by jaaacki** [comment]
getting the same error

**Comment by PeterHagen** [comment]
It seems the error depends on the response of the LLM. I did upgrade to Ollama 0.3.0 today. Before, I think it happened every time, but I can't pinpoint at the moment when this occurs.

**Comment by PeterHagen** [comment]
Unfortunately, the issue occurs in Chrome also.

**Comment by PeterHagen** [comment]
A few days ago I deleted all my chats. Since then I didn't have this issue anymore.
